deno_dist/helper/ssg/index.ts

```diff
@@ -122,11 +121,7 @@ export const fetchRoutesContent = async <
   const htmlMap = new Map<string, { content: string | ArrayBuffer; mimeType: string }>()
   const baseURL = 'http://localhost'
 
-  for (const route of inspectRoutes(app)) {
-    if (route.isMiddleware) {
-      continue
-    }
-
+  for (const route of filterStaticGenerateRoutes(app)) {
```

Comment:
@watany-dev 

Do you mean removing this line?

Reply:
@yusukebe 

Since the filter is filtered by ismiddleware conditions in filterStaticGenerateRoutes, there is no need to check it again here.